packages/@aws-cdk/toolkit/lib/toolkit/toolkit.ts

```diff
@@ -47,6 +47,13 @@ export interface ToolkitOptions {
    */
   ioHost?: IIoHost;
 
+  /**
+   * Allow emojis in messages sent to the IoHost.
+   *
+   * @default true
+   */
+  emojis?: boolean;
```

Comment:
ticket says `emojis` so that's what i went with but i slightly prefer `allowEmojis`...